### PR TITLE
hlaminer 1.3.0

### PR DESCRIPTION
**Pull request by warrenlr**
### Have you:
- [ ] Followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-science/blob/master/.github/CONTRIBUTING.md) document?
- [ ] Checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-science/pulls) for the same update/change?
- [ ] Run `brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting) and corrected all errors?
- [ ] Built your formula locally prior to submission with `brew install <formula>`?
### New formula: have you
- [ ] Written a sensible test? (The best test is to compile and run a sample program.)
### Updates to existing formula: have you
- [ ] Removed the `revision` line, if any, when bumping the version number?
- [ ] Added/bumped the `revision` number if the changes affect the precompiled bottles?
- [ ] Not altered the `bottle` section?
- [ ] Checked that the tests still pass?

.sh in ./bin are all executables.  Same with blast and formatdb. The .txt is a config file that is important at that location

### CODE REVIEW

File: hlaminer.rb [hunk 1]
```diff
@@ -4,13 +4,14 @@ class Hlaminer < Formula
   # doi "10.1186/gm396"
   # tag "bioinformatics"
 
-  url "http://www.bcgsc.ca/platform/bioinfo/software/hlaminer/releases/1.1/HLAminer.tar.gz"
-  version "1.1.0"
-  sha256 "2c3e4458215f06764f58e8f3d95fb8e3a47d2a19152478d047453944c8cb84c3"
+  url "http://www.bcgsc.ca/platform/bioinfo/software/hlaminer/releases/1.3.1/HLAminer_v1-3-1.tar.gz"
+  version "1.3.0"
+  sha256 "a009575d80484c7c1dd188f29c21419ccca3ff7feea31128d0906cfc944959fe"
   revision 1
```
Comment (jonchang) [inline hlaminer.rb]: You don't need this line if you are changing the `version` number.

File: hlaminer.rb [hunk 2]
```diff
@@ -4,13 +4,14 @@ class Hlaminer < Formula
...
 
   bottle do
     cellar :any_skip_relocation
+    ###need to update those:
```
Comment (jonchang) [inline hlaminer.rb]: You can remove this comment; the test bot will update them automatically.

File: hlaminer.rb [hunk 3]
```diff
@@ -29,6 +30,6 @@ def install
   end
 
   test do
-    assert_match "HLAminer.pl [v1.1]", shell_output("#{bin}/HLAminer.pl", 255)
+    assert_match "HLAminer.pl [v1.3]", shell_output("#{bin}/HLAminer.pl", 255)
```
Comment (jonchang) [inline hlaminer.rb]: It looks like the are some example files in the `test-demo` folder? It would be nice to have a test that actually ran the program with input and output.

**Comment by jonchang** [comment]
Thanks for your update! Once the comments in the diff are address this pull request should be ready to go.

File: hlaminer.rb (hunk 3 above)
Comment (warrenlr) [inline hlaminer.rb]: There is a test that runs the program, as indicated in the README-FIRST.txt

**Comment by warrenlr** [comment]
just to confirm, no further edits needed on my end?

File: hlaminer.rb (hunk 3 above)
Comment (warrenlr) [inline hlaminer.rb]: just committed to patch-21, no need to do another pull request?

File: hlaminer.rb (hunk 1 above)
Comment (jonchang) [inline hlaminer.rb]: Please remove the `revision 1` line.

File: hlaminer.rb (hunk 3 above)
Comment (jonchang) [inline hlaminer.rb]: Right, so since the test is included the `test do` block should execute that test so we know that the program is installed correctly.

**Comment by jonchang** [comment]
Please fix this audit error:

```
==> brew audit homebrew/science/hlaminer
Error: 1 problem in 1 formula
homebrew/science/hlaminer:
  * 'revision 1' should be removed
```

**Comment by jonchang** [comment]
This is still failing audit checks. Please run `brew audit --strict --online hlaminer` and fix any errors that pop up.

**Comment by warrenlr** [comment]
the command returns no errors on my system:

rwarren02:coverage rwarren$ which brew
/Users/rwarren/Linuxbrew/Homebrew-linuxbrew-7ea7503/bin//brew
rwarren02:coverage rwarren$ brew audit --strict --online hlaminer
rwarren02:coverage rwarren$

**Comment by apjanke** [comment]
@BrewTestBot test this please

**Comment by jonchang** [comment]
@BrewTestBot test this please

**Comment by jonchang** [comment]
@BrewTestBot test this please

**Comment by jonchang** [comment]
Merged with audit fixes.

**Comment by sjackman** [comment]
@sjackman :penguin:

File: hlaminer.rb [hunk 4]
```diff
@@ -4,10 +4,10 @@ class Hlaminer < Formula
   # doi "10.1186/gm396"
   # tag "bioinformatics"
 
-  url "http://www.bcgsc.ca/platform/bioinfo/software/hlaminer/releases/1.1/HLAminer.tar.gz"
-  version "1.1.0"
-  sha256 "2c3e4458215f06764f58e8f3d95fb8e3a47d2a19152478d047453944c8cb84c3"
-  revision 1
+  url "http://www.bcgsc.ca/platform/bioinfo/software/hlaminer/releases/1.3.1/HLAminer_v1-3-1.tar.gz"
+  version "1.3.0"
```
Comment (sjackman) [inline hlaminer.rb]: @jonchang @warrenlr Why is this formula tagged as version 1.3.0 when the file appears to be version 1.3.1?

**Comment by jonchang** [comment]
Looks like the version changed at some point and I definitely didn't notice
that. Can you open a PR to fix it? I won't be able to get back to a
computer until tomorrow.

**Comment by sjackman** [comment]
Can do.

**Comment by sjackman** [comment]
Done. https://github.com/Homebrew/homebrew-science/pull/4677
